frontend/app/src/components/NavigationMenu.vue

```diff
@@ -219,146 +219,64 @@ const navItems: MenuItem[] = [
 </script>
 
 <template>
-  <div>
-    <VList
-      nav
-      class="navigation-menu"
-      :class="{ 'navigation-menu--mini': isMini }"
-    >
-      <VListItemGroup>
-        <template v-for="(navItem, i) in navItems">
-          <VListItem
-            v-if="navItem.type === 'item'"
-            :key="i"
-            :class="`navigation-menu__item navigation__${navItem.class}`"
-            active-class="navigation-menu__item--active"
-            :to="navItem.route"
+  <div class="p-2" :class="{ '!p-0': isMini }">
+    <template v-for="(navItem, i) in navItems">
+      <RouterLink
+        v-if="navItem.type === 'item'"
+        :key="i"
+        #default="{ isActive, href }"
+        :to="navItem.route"
+      >
+        <a :href="href">
```

Comment:
do we need both `Routerlink` and `a` here?

Reply:
Yes because I need to use the default template, to get the `isActive`

Reply:
Oh ok and the default template if you don't set it has an `a` in the slot

Reply:
yes exactly